### ISSUE TITLE
Bug when trying to disburse milestone

### ISSUE THREAD

**Issue by Kisgus** [Giveth/giveth-dapp#621]
We get the same error message in the Venezuela Campaign as in  https://github.com/Giveth/giveth-dapp/issues/445 when trying to "disburse" funds

**Milestone 1**

https://beta.giveth.io/campaigns/5c0e84ea06392d786b64d484/milestones/5c0e959806392d786b64d48c

**Milestone 2**

https://beta.giveth.io/campaigns/5c0e84ea06392d786b64d484/milestones/5c0e959806392d786b64d48c

**Comment by Kisgus** [comment]
Thanks for looking into this @perissology, do you have any fix in mind?

**Comment by perissology** [comment]
yes, it's fixed but hasn't been released yet. There were some big code changes that this fix got made on top of. So we're still testing. Hopefully within the next few days.

Unfortunately there is a max of 8 donations that can be withdrawn at a time, so you will have to "withdraw" multiple times to get all the funds.

I'm not sure if you're a dev or not. You can always run this repo locally, and set the `.env.local` file `environment=beta` and it will run the latest codebase against the beta data.

**Comment by Kisgus** [comment]
Okay thanks. Not really a dev, so I will just be waiting. Is it correctly understood once this release has been merged I will be able to disburse? Alternatively if I had the option in the UI to just disburse 8 tx at the time that could also do the trick on the short time if I could manually just disburse it a few times.

**Comment by perissology** [comment]
you will only be able to disburse 8 at a time. It is a current limitation w/ the smart contracts

**Comment by Kisgus** [comment]
So if that option of "disburse 8" would be in the UI that would be totally okay without contract updates.
Right now I only have the option of "disbursing" in the UI, which tries to disburse all and therefore fails.

**Comment by perissology** [comment]
yep, it will only disburse max 8 & show a note that you will need to disburse again

**Comment by vojtechsimetka** [comment]
After the latest UI release the disburse button is no longer there even though there are still some donations to be disbursed.

![screen shot 2019-01-22 at 12 39 47 pm](https://user-images.githubusercontent.com/7974813/51534529-bec8d700-1e24-11e9-8449-8b7caa486b3e.png)

**Comment by mroberts10** [comment]
Hi @Kisgus can we close this issue now?

**Comment by Kisgus** [comment]
We are now trying to disburse, seems to work, will confirm when/if funds has been successfully disbursed :)

**Comment by GriffGreen** [comment]
This is still an issue it seems
https://beta.giveth.io/campaigns/5cfe3366228b71296a491fa7/milestones/5d1ba874bd45530e23ad86ed
In this milestone there is a bunch of small donations
<img width="1290" alt="Screen Shot 2019-07-15 at 11 23 10 PM" src="https://user-images.githubusercontent.com/14945613/61250288-a09abe80-a757-11e9-85aa-dbde0f897922.png">
We need a more robust way to deal with lots of small donations in one go... I think a pop up similar to the warning that there will be 2 donations

**Comment by arbreton** [comment]
Reading through this issue now

**Comment by GriffGreen** [comment]
@aminlatifi this is related to #871

**Comment by GriffGreen** [comment]
this still doesnt work.... we need to do some sort of individual hack for https://beta.giveth.io/campaigns/5cfe3366228b71296a491fa7/milestones/5d1ba874bd45530e23ad86ed

**Comment by arbreton** [comment]
I applied a fix that should allow collecting at 5 per transaction, it should work now! please keep me posted @GriffGreen
